app/src/main/java/org/fossasia/openevent/app/data/db/configuration/OrgaDatabase.java

```diff
@@ -37,4 +45,26 @@ public void onPreMigrate() {
             addColumn(SQLiteType.TEXT, "url");
         }
     }
+
+    /**
+     * Sets foreign key constraints action onDelete to CASCADE for Attendee, SocialLink and Ticket models.
+     * Dropping all the tables.
+     */
+    @Migration(version = 3, database = OrgaDatabase.class)
+    public static class MigrationTo3 extends BaseMigration {
+
+        @Override
+        public void migrate(@NonNull DatabaseWrapper db) {
+            List<String> tables = new ArrayList<>();
+            Cursor c = db.rawQuery("SELECT name FROM sqlite_master WHERE type='table'", null);
+
+            while (c.moveToNext())
+                tables.add(c.getString(0));
+
+            for (String table : tables) {
+                String dropQuery = "DROP TABLE IF EXIST " + table;
+                db.execSQL(dropQuery);
+            }
+        }
```

Comment:
You can just delete the database using util model, no need to do recursive query